indy_node/test/txn_author_agreement/test_recover_taa_from_ledger.py

```diff
@@ -106,11 +106,24 @@ def test_recover_taa_from_ledger(txnPoolNodeSet,
     ensure_all_nodes_have_same_data(looper, txnPoolNodeSet)
 
     # Step 13. Retire TAA written using old handler, make sure ratification date is not spoiled
-    sdk_send_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet_trustee, version_1, retired=retired_time)
+    sdk_send_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet_trustee, version_1, retired=retired_time_in_past)
     res_1 = sdk_get_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet_trustee, version=version_1)[1]
+    check_new_format_taa(res_1, version_1, taa_1_ratification_ts, retired_time_in_past)
 
-    assert TXN_AUTHOR_AGREEMENT_DIGEST in res_1['result']['data']
-    assert TXN_AUTHOR_AGREEMENT_RATIFICATION_TS in res_1['result']['data']
-    assert res_1['result']['data'][TXN_AUTHOR_AGREEMENT_VERSION] == version_1
-    assert res_1['result']['data'][TXN_AUTHOR_AGREEMENT_RETIREMENT_TS] == retired_time
-    assert res_1['result']['data'][TXN_AUTHOR_AGREEMENT_RATIFICATION_TS] == get_txn_time(res_0['result'])
+    # Step 14. Disable TAAs written using old handler, make sure ratification date is not spoiled
+    disable_res = sdk_send_txn_author_agreement_disable(looper, sdk_pool_handle, sdk_wallet_trustee)[1]
+
+    res_0 = sdk_get_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet_trustee, version=version_0)[1]
+    check_new_format_taa(res_0, version_0, taa_0_ratification_ts, get_txn_time(disable_res['result']))
+
+    res_1 = sdk_get_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet_trustee, version=version_1)[1]
+    check_new_format_taa(res_1, version_1, taa_1_ratification_ts, retired_time_in_past)
+
+
+def check_new_format_taa(result, version, ratification_ts, retired_time=None):
```

Comment:
If we create helper function for checking something I'd prefer have more clear name, like `check_result_contains_expected_taa_data`. Another possibility is to create function like this:
```
def expected_taa_data(version, ratification_ts, ...):
   result = {
      TXN_AUTHOR_AGREEMENT_VERSION: version,
      ...
   }
   ...
   return result
```
and then use it like this:
```
assert result['result']['data'] == expected_taa_data(version, ratification_ts, ...)
```